header.go

```diff
@@ -344,6 +344,18 @@ func (h *ResponseHeader) SetContentEncodingBytes(contentEncoding []byte) {
 	h.contentEncoding = append(h.contentEncoding[:0], contentEncoding...)
 }
 
+// AddVaryBytes add value to the 'Vary' header if it's not included
+func (h *ResponseHeader) AddVaryBytes(value []byte) {
```

Comment:
This is such a specific function and our API surface is already huge. Lets not export this function.

Reply:
My thoughts was, it can also be used to add other values into 'Vary' header in application code.
Any way, it's ok to me to change it to 'addVaryBytes'